README.md

```diff
@@ -10,13 +10,27 @@ REST Book is a Visual Studio Code extension that allows you to perform REST call
 - Organize multiple REST Calls within one file.
 - Intermingle markdown for documenting your calls.
 - View rich HTML and image responses directly inside the Notebook.
-- Basic Authentication
-- Use data from one call in the next
+- Optionally use the Interactive Window interface.
+- Basic Authentication.
+- Use data from one call in the next.
 - Save API keys and other sensitive information outside of the Notebook to use securely in calls.
 
 ## Requirements
 
-Must be using the latest version of Visual Studio Code Insiders edition.
+This extension uses [proposed API](https://code.visualstudio.com/api/advanced-topics/using-proposed-api#sharing-extensions-using-the-proposed-api) so you must be using the latest version of Visual Studio Code Insiders edition.
+
+Either launch it from the command line:
+```
+code-insiders . --enable-proposed-api=tanhakabir.rest-book
+```
+
+or use the `Preferences: Configure Runtime Arguments` to add this extension's ID to the `enable-proposed-api` array, then relaunch:
+```json
+{
+    ...
+    "enable-proposed-api": ["tanhakabir.rest-book"]
+}
+```
```

Comment:
@tanhakabir the extra information I added above seems to be redundant. I just verified that I was able to install 5.0.1 from Marketplace into 1.64 and us it without doing anything special to allow the proposed API. Maybe you've been able to publish the extension despite its "enabledApiProposals" clause in package.json. In my extension deleopment experience that's always barred publication.

Reply:
Thank you! Sorry I had missed this!